src/NBomber/DomainServices/Reporting/HtmlReport.fs

```diff
@@ -46,9 +48,9 @@ module NumberReqChart =
 
 module StatisticsTable =    
 
-    let print (assets: Assets, scnStats: ScenarioStats[]) =     
+    let print (assets: Assets, scnStats: ScenarioStats[], assertResults: Result<Assertion,DomainError>[]) =     

```

Comment:
maybe we should receive already filtered errors since we are going to print only errors
instead of Result<Assertion,DomainError>[] we could receive just DomainError[]